deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -80,93 +102,101 @@ spec:
       - kind: Services
         name: ""
         version: v1
+      - kind: ServiceMonitors
```

Comment:
This was recommended to be added by the `operator-sdk scorecard` run since we're creating `ServiceMonitor` objects as part of the deployment.